Sources/Views/PlayButtonView.swift

```diff
@@ -28,7 +28,7 @@ open class PlayButtonView: UIView {
 
     // MARK: - Properties
 
-    open let triangleView = UIView()
+    public let triangleView = UIView()
```

Comment:
“open” is now simply “more public than public”, providing a very simple and clean model.